03_letters/srini/main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+)
+
+var out io.Writer = os.Stdout
+
+func letters(s string) map[rune]int {
+	letterMap := make(map[rune]int)
+
+	for _, r := range s {
+		if _, ok := letterMap[r]; ok {
+			letterMap[r]++
+		} else {
+			letterMap[r] = 1
+		}
+	}
+	return letterMap
+}
+
+func print(m map[rune]int) {
```

Comment:
Sorry to request a change here, but I've updated the [lab description](https://gotraining.appspot.com/gotraining.slide#43) for this to be:
```go
func sortLetters(m map[rune]int) []string 
```
which should make it more testable.
Apologies for the extra effort!

Reply:
Your simulation of a real user, @juliaogris, is spot on. Changing requirements after the code has been written :smile_cat: